src/main/java/net/minecraftforge/common/ForgeHooks.java

```diff
@@ -1386,4 +1386,9 @@ public static void onAdvancement(EntityPlayerMP player, Advancement advancement)
     {
         MinecraftForge.EVENT_BUS.post(new AdvancementEvent(player, advancement));
     }
+
+    public static float getPlayerReachDistance(EntityPlayer player) {
```

Comment:
brace on new line

Reply:
Ah.. gets me every time. Thanks for catching it.